software-engineering-policies/Containers/ContainerPolicy.md

```diff
@@ -15,6 +15,24 @@ Containers are in use in Data Science and Software Engineering, in all environme
 * [Container best practices](./ContainerBestPractices.md) should be adopted by all teams using containers.
 * The Lead Software Engineer for each team is responsible for ensuring that this policy is adhered.
 
+## Security
+
+Use of containers provides a convinient way for engineering teams to develop and deploy applications in a robust way without the overhead of always having to acomodate the hosting platform. However this can come at a price due to security being shifted more towards the development team rather than the operations teams. To ensure we are doing the best we can when it comes to containerisation safety, the following should be adhered to.
+
+### Container Image
+
+The container image being selected should be the leanest image for what you require. This means using an base image that contains the minimum amount of dependencies required to run your application. What this does is reduce the attack surface a threat actor has to perform malicious actions within your service. It also has the added benefit of reduced container image size as we are not holding dependencies we are not using.
```

Comment:
Is it worth adding a note to recommend the use of a multistage docker file to ensure all build/test tools that are needed to build but not run the application are stripped from the final image?